tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,21 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+#@pytest.mark.skip
 def test_removing_not_found_is_false():
+    #Arrange
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
-
+    #Act
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    #Assert
+    assert result == False
+    assert "a" in vendor.inventory
+    assert "b" in vendor.inventory
+    assert "c" in vendor.inventory
```

Comment:
Nice work on explicitly checking these things! One thing that you could add is an assert statement that checks to see if `item` is **not** inside of the inventory as well. 
```suggestion
    assert result == False
    assert "a" in vendor.inventory
    assert "b" in vendor.inventory
    assert "c" in vendor.inventory
    assert item not in vendor.inventory
```